src/View/layout.html.twig

```diff
@@ -25,7 +25,7 @@
 			{# ------------------- Navbar Desktop ----------------------#}
 			<nav class="nav container-fluid d-flex h-auto w-auto d-none d-md-block sticky-top pe-0">
 				<div class="d-flex h5 Bootstraps heading text-white align-items-center ">
-					<a class="navbar-brand d-flex align-items-center ms-2 " href="/">
+					<a class="navbar-brand d-flex align-items-center ms-2">
```

Comment:
No a `<a>` without an href is not allowed. Either you let the href, or you choose another tag that a `<a>` (`<span>`, `<p>`, `<div>` )